sourcemash/categorize.py

```diff
@@ -1,7 +1,9 @@
 from collections import Counter
 from string import punctuation
 
-def categorizeItem(title, text, category_dict):
+def categorize_item(title, text, category_dict):
```

Comment:
Would it make sense to move this to the scraper? It feels weird passing around our category_dict.